wpilibc/src/main/native/include/frc/Alert.h

```diff
@@ -88,6 +83,11 @@ class Alert {
    */
   Alert(std::string_view group, std::string_view text, AlertType type);
 
+  Alert(Alert&&);
+  Alert& operator=(Alert&&);
+
+  ~Alert();
```

Comment:
You might need to explicitly delete the copy constructor and copy assignment?

Reply:
Implicitly defined copy constructors are deleted when a move constructor is declared but I can toss in an explicit delete to make that clear